backend/business-partner-agent/src/main/java/org/hyperledger/bpa/impl/verification/prooftemplates/ValidAttributeConditionOperator.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2020-2021 - for information on the respective copyright owner
+ * see the NOTICE file and/or the repository at
+ * https://github.com/hyperledger-labs/business-partner-agent
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.hyperledger.bpa.impl.verification.prooftemplates;
+
+import javax.validation.Constraint;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Constraint(validatedBy = {})
+public @interface ValidAttributeConditionOperator {
+    String message() default "{validatedValue} is an unsupported condition operator for proof template attributes";
```

Comment:
I added a default resource bundle for localization. Is it possible to load this message from there? See BPAMessageSource

Reply:
It's not possible, because annotation default values have to be constants, because annotation values are evaluated at compile time.